vite.config.js

```diff
@@ -3,6 +3,9 @@ import react from '@vitejs/plugin-react-swc'
 
 // https://vitejs.dev/config/
 export default defineConfig({
+  optimizeDeps: {
+    include: ['esm-dep > cjs-dep'],
```

Comment:
if it is important, better to document it in readme.md

Reply:
not needed i removed it , i had a temporary bug and this fixed it at the time, but it's no longer needed